Customization/T200/PhoneRepairShop/_project/File_Bin_PhoneRepairShop_Code_dll.xml

```diff
@@ -0,0 +1 @@
+﻿<File AppRelativePath="Bin\PhoneRepairShop_Code.dll" FileID="0d1e59e0-0d1f-4e0b-818f-760417aa80c5" />
```

Comment:
It seems that the project now should as well contain File items for the csv files from InputData.